daemon/k8s_watcher.go

```diff
@@ -35,7 +35,6 @@ import (
 	"github.com/cilium/cilium/pkg/k8s/apis/cilium.io"
```

Comment:
Will the error example in your commit be logged for every single CNP NodeStatus update? If this is the case, this may alarm some users since there will be a lot of errors on the K8s apiserver :(

Reply:
Yes